envbuilder.go

```diff
@@ -404,9 +404,29 @@ func Run(ctx context.Context, options Options) error {
 		util.AddToDefaultIgnoreList(util.IgnoreListEntry{
 			Path:            ignorePath,
 			PrefixMatchOnly: false,
+			AllowedPaths:    map[string]struct{}{},
```

Comment:
Now that I think of it, all of that is just to avoid a panic on assigning to a nil map. However, using a `[]string` instead doesn't have that issue, and might even be a tiny bit faster.